.gitignore

```diff
@@ -234,6 +234,7 @@ _Pvt_Extensions
 
 # Tools
 tools/
+tools/nuget/nuget.exe
```

Comment:
We don't currently use a proper bootstrapper to download the initial `NuGet.exe` and instead rely on the existence of any version (which is updated to latest on each build). See https://github.com/reactiveui/ReactiveUI/blob/develop/build.cmd#L2

👍 to potentially moving to using the cake bootstrapper (proper) but downsides include if someone wants to build ReactiveUI and they are behind a corporate firewall then the bootstrapper will fail. I went with embedding the NuGet.exe binary into the repository because I'm often in such situations.